src/components/modules/blockManager.ts

```diff
@@ -624,6 +624,15 @@ export default class BlockManager extends Module {
     });
   }
 
+  /**
+   * Remove drop targets style from all Blocks.
+   */
+  public clearDropTargets(): void {
+    this.blocks.forEach((block) => {
+      block.dropTarget = undefined;
```

Comment:
That looks unintuitive for me, could it be `false` here instead of `undefined`?

Reply:
done